### PR TITLE
do not send events to the event busy unless datacite_crossref

### PR DESCRIPTION
**Pull request by wendelfabianchinsamy**
## Purpose

closes: https://github.com/datacite/levriero/issues/190

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: spec/models/related_identifier_spec.rb [hunk 1]
```diff
@@ -104,9 +104,28 @@
         allow(Rails.logger).to receive(:info)
 
         expect(RelatedIdentifier.push_item(item)).to eq(1)
-        expect(Maremma).to have_received(:post).twice
+        expect(Maremma).to have_received(:post).with("https://fake.lagattino.com/events", anything).once
 
         expect(Rails.logger).to have_received(:info).with("[Event Data] https://doi.org/10.1234/example example_type https://doi.org/10.5678/related pushed to Event Data service.")
+      end
+
+      it "does not push the event to the event data bus when source_id is not datacite_crossref" do
```
Comment (digitaldogsbody) [inline spec/models/related_identifier_spec.rb:112]: I think this test is misnamed, as it checks that the event gets posted to _both_ the internal Event Data on Lupo and also to the Bus. Not even anything close to a blocker, but fixing it now might make thins easier in the future :sweat_smile:

**Review by digitaldogsbody — APPROVED** [review]
Looks great, thanks Wendel!

File: spec/models/related_identifier_spec.rb (hunk 1 above)
Comment (wendelfabianchinsamy) [reply spec/models/related_identifier_spec.rb]: Good catch @digitaldogsbody. I've updated the pr 👍🏽

**Review by digitaldogsbody — APPROVED** [review]

**Comment by digitaldogsbody** [comment]
I made one other small change to the test description, but now I can't approve because I am the last committer :melting_face:

**Comment by wendelfabianchinsamy** [comment]
> I made one other small change to the test description, but now I can't approve because I am the last committer 🫠

Lol thanks Mike. Mondays, am i right 🤣

**Review by digitaldogsbody — APPROVED** [review]

**Pull request merged**